test/controllers/comments_controller_test.rb

```diff
@@ -36,7 +36,7 @@ class CommentsControllerTest < ActionDispatch::IntegrationTest
   test 'user should be able to edit their own comments' do
     user = create(:user)
     curiosity = create(:curiosity_card)
-    comment = create(:comment, curiosity_card: curiosity, user: )
+    comment = create(:comment, curiosity_card: curiosity, user: user)
```

Comment:
This changes has nothing to do with ruby version downgrade hehe

Reply:
actually, we need it because it's a new release from ruby 3.1.0 https://www.ruby-lang.org/en/news/2021/11/09/ruby-3-1-0-preview1-released/ hehe and my test was broken because of it :c hehe

Reply:
Holy smokes!
Using `user:` does actually looks for a `user` var. That's crazy dude haha